casual/casual-inbound-handler-fielded-buffer/src/main/java/se/laz/casual/jca/inbound/handler/buffer/fielded/FieldedBufferHandler.java

```diff
@@ -31,24 +32,26 @@ public boolean canHandleBuffer(String bufferType)
     }
 
     @Override
-    public ServiceCallInfo fromRequest(Proxy p, Method method, InboundRequest request)
+    public ServiceCallInfo fromRequest(InboundRequest request, InboundRequestInfo requestInfo)
     {
         Object[] params;
-        if( methodAccepts( method, request ) )
+        Method proxyMethod = requestInfo.getProxyMethod().orElseThrow(() -> new InboundRequestException("Missing proxy method, requestInfo: " + requestInfo));
+        Method realMethod =  requestInfo.getRealMethod().orElseThrow(() -> new InboundRequestException("Missing real method, requestInfo: " + requestInfo));
+        if( methodAccepts( proxyMethod, request ) )
```

Comment:
If we have the real, why would we use the proxy in these accepts methods? Couldn't we just retrieve the real and pass that through, otherwise pass the proxy?